upgrading/v3.1-to-v3.2.md

```diff
@@ -0,0 +1,193 @@
+- Combine tags for cross-cutting concerns (like delivery methods)
+
+*See: [Enhanced Tags](../specification/tags) for comprehensive documentation*
+
+### Advanced HTTP Method Support
+
+- **QUERY method** for complex filtering with request bodies
+- **Custom HTTP methods** via additionalOperations
+- **Enhanced parameter handling** including querystring location
+
+*See: [HTTP Methods](../specification/http-methods) for implementation details*
+
+### Sequential and Streaming Data
+
+- **Server-Sent Events** (text/event-stream)
+- **JSON streaming** (application/jsonl, application/json-seq)
+- **Enhanced multipart** handling
+- **itemSchema** field for describing sequential data
+
+*See: [Sequential Media Types](../specification/media-types) for complete guide*
+
+### Security Enhancements
+
+- **OAuth2 Device Authorization** flow for limited-input devices
+- **Enhanced metadata** support with oauth2MetadataUrl
+- **Deprecation marking** for security schemes
+- **URI-based references** for security schemes
+
+*See: [Security](../specification/security) for updated authentication methods*
+
+### Better Examples and Documentation
+
+- **Structured examples** with dataValue field
+- **Serialization examples** with serializedValue field
+- **Enhanced servers** with name fields
+- **Document identity** with $self field
+
+*See: [Providing Documentation and Examples](../specification/docs) for enhanced example handling*
+
+## Upgrade Benefits
+
+## Gradual Enhancement Strategy
+
+Consider adopting new features incrementally:
+
+1. **Enhance server descriptions** - Add `name` fields to server objects
+2. **Improve examples** - Use `dataValue` and `serializedValue` for clearer examples
+3. **Add document identity** - Include `$self` field if URI resolution clarity is needed
+1. **Tag enhancements** - Convert `x-displayName` to `summary`, `x-tagGroups` to tag hierarchy
+2. **Remove custom extensions** - Replace with native 3.2 functionality where available
+1. **New HTTP methods** - Implement QUERY method for complex filtering scenarios
+2. **Streaming support** - Add sequential media types for real-time APIs
+3. **Enhanced security** - Implement OAuth2 device flow if relevant to your use cases
+
+## Migration Checklist
+
+### Essential Steps
+
+- [ ] Update `openapi` version to `3.2.0`
+- [ ] Validate existing specification with OpenAPI 3.2 compatible tools
+
+### Gradual Enhancements (As Needed)
+
+- [ ] Replace custom extensions with native 3.2 features
+- [ ] Add `name` fields to server objects for better identification
+- [ ] Enhance examples with `dataValue`/`serializedValue` fields
+- [ ] Consider tag hierarchy for better API organisation
+- [ ] Evaluate new HTTP methods for relevant use cases
+- [ ] Review security schemes for OAuth2 device flow applicability
+- [ ] Consider sequential media types for streaming or real-time APIs
+
+## Tools and Resources
+
+- [OpenAPI 3.2 Specification](https://spec.openapis.org/oas/latest) (when available)
```

Comment:
```suggestion
- [OpenAPI 3.2 Specification](https://spec.openapis.org/oas/v3.2) (when available)
```
This will link to the latest 3.2 patch release and not to the latest major.minor release.